manifests/1.14/deploy/vsphere-csi-controller-ss.yaml

```diff
@@ -15,27 +17,43 @@ spec:
         app: vsphere-csi-controller
         role: vsphere-csi
     spec:
-      serviceAccount: vsphere-csi-controller
+      serviceAccountName: vsphere-csi-controller
       nodeSelector:
         node-role.kubernetes.io/master: ""
       tolerations:
-      - key: node.cloudprovider.kubernetes.io/uninitialized
-        value: "true"
-        effect: NoSchedule
-      - key: node-role.kubernetes.io/master
-        effect: NoSchedule
+        - operator: "Exists"
+          key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      hostNetwork: true
       containers:
+        - name: csi-attacher
+          image: quay.io/k8scsi/csi-attacher:v1.1.1
+          args:
+            - "--v=4"
+            - "--timeout=60s"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          volumeMounts:
+            - mountPath: /csi
+              name: socket-dir
         - name: vsphere-csi-controller
-          image: gcr.io/cloud-provider-vsphere/csi/release/driver:v0.2.1
+          image: cloudnativestorage/vsphere-block-csi-driver:latest
```

Comment:
Thanks for catching this.

We have released GA images at `vmware/vsphere-block-csi-driver:v1.0.0` and `vmware/volume-metadata-syncer:v1.0.0`.  Let me replace them for now, to keep in-sync with official documentation. - https://docs.vmware.com/en/VMware-vSphere/6.7/Cloud-Native-Storage/GUID-039425C1-597F-46FF-8BAA-C5A46FF10E63.html

We can change the image location to `gcr.io/cloud-provider-vsphere/csi/release/driver:<new-tag>`, when new images are available from this location after we cut a new tag release on `vsphere-csi-driver` repository.

We can then change official vSphere documentation to use YAML files directly from this repository with github URL, instead of asking user to copy/paste from documentation.